src/util/cmdlineargs.cpp

```diff
@@ -148,6 +166,10 @@ void CmdlineArgs::printUsage() {
                         info     - Above + Informational messages\n\
                         debug    - Above + Debug/Developer messages\n\
                         trace    - Above + Profiling messages\n\
+\n\
+--logFlushLevel LEVEL   Defines which messages level causes flushing all\n\
```

Comment:
Consider rephrasing. The term "messages" appears twice in this sentence and the grammar is broken.